useragent.gemspec

```diff
@@ -8,7 +8,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["LICENSE", "README.md", "lib/**/*.rb"]
 
-  s.add_development_dependency "rake", "~> 10.0"
+  s.add_development_dependency "rake", "~> 13.0"
```

Comment:
Rake version 10 doesn't seem compatible with Ruby 3.2. Let's bump it up to the latest version. 13 is compatible with all Rubies in the CI matrix.